.github/workflows/coverity.yml

```diff
@@ -0,0 +1,46 @@
+# Based on https://github.com/ruby/actions-coverity-scan/blob/master/.github/workflows/coverity-scan.yml
+name: coverity
+on:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Could we instead trigger scans on pull requests?  They're not very frequent so I don't expect issues due to throttling.  I'd rather address issues in the pull request before it gets merged.

Reply:
We can, but it has a few pain points:

- Will need to sleep & poll in a loop in the script until scan results become available
- I don't know yet how to query scan results via HTTP api - it looks like it should be possible, but not obvious.
- Something needs to be done about the baseline - i.e. right now coverity reports quite many issues, so those need to be either fixed or marked as legacy for PR check to only report new issues.

Reply:
I can take care of the baseline.  I'll fix the bugs and mark the false positives.  Do you have a link to some small howto?  I'd also need to run the checks manually until this gets merged.